src/legacy/core_plugins/vis_default_editor/public/components/controls/time_interval.tsx

```diff
@@ -56,15 +58,45 @@ function TimeIntervalParamEditor({
   let selectedOptions: ComboBoxOption[] = [];
   let definedOption: ComboBoxOption | undefined;
   let isValid = false;
+  const errors = [];
   if (value) {
     definedOption = find(options, { key: value });
     selectedOptions = definedOption ? [definedOption] : [{ label: value, key: 'custom' }];
-    isValid = !!(definedOption || isValidInterval(value, timeBase));
+    isValid = !!definedOption || isValidInterval(value, timeBase);
```

Comment:
`isValidInterval` returns `true` for the case `14d`. But the app sends `2w` in the request to elasticsearch. That's why we additionally check the elasticsearch expression for validity in line 82.